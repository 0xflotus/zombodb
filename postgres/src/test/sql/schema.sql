CREATE SCHEMA test_schema;
CREATE TABLE test_schema.test_table (
  id    SERIAL8 PRIMARY KEY,
  title TEXT
);
CREATE INDEX idxtest_table ON test_schema.test_table USING zombodb(zdb(test_table), zdb_to_json(test_table)) WITH (url='http://localhost:9200/');
SELECT * FROM zdb_estimate_count('test_schema.test_table', 'beer');

DROP SCHEMA test_schema CASCADE;
