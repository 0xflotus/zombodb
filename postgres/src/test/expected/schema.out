CREATE SCHEMA test_schema;
CREATE TABLE test_schema.test_table (
  id    SERIAL8 PRIMARY KEY,
  title TEXT
);
CREATE INDEX idxtest_table ON test_schema.test_table USING zombodb(zdb(test_table), zdb_to_json(test_table)) WITH (url='http://localhost:9200/');
SELECT * FROM zdb_estimate_count('test_schema.test_table', 'beer');
 zdb_estimate_count 
--------------------
                  0
(1 row)

DROP SCHEMA test_schema CASCADE;
NOTICE:  drop cascades to table test_schema.test_table
